src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj

```diff
@@ -80,6 +80,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="DotNet.ReproducibleBuilds" Version="1.1.1">
```

Comment:
Let's only add [`Microsoft.SourceLink.GitHub`](https://www.nuget.org/packages/Microsoft.SourceLink.GitHub/). 

We don't need the other dependencies included  in `Dotnet.ReproducableBuilds`.